spot_driver/spot_driver/spot_ros2.py

```diff
@@ -428,7 +433,7 @@ def __init__(self, parameter_list: Optional[typing.List[Parameter]] = None, **kw
         has_arm = self.mock_has_arm
         if self.spot_wrapper is not None:
             has_arm = self.spot_wrapper.has_arm()
-        if has_arm:
+        if has_arm and not self.gripperless:
             all_cameras.append("hand")
         self.declare_parameter("cameras_used", all_cameras)
```

Comment:
I think this param is not used at all in this node and maybe can be completely deleted